src/state_manager/vm_circ_supply.rs

```diff
@@ -107,6 +111,89 @@ impl GenesisInfo {
             fil_reserve_disbursed,
         })
     }
+
+    pub fn get_circulating_supply<DB: Blockstore>(
```

Comment:
How come this is not just `get_vm_circulating_supply().fil_circulating`?

Reply:
This endpoint can be heavy in processing; I wonder if this must be publicly available.

Reply:
Let's write this information in a comment.